src/main/java/org/openrewrite/java/dropwizard/method/RemoveUnnecessaryOverride.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.dropwizard.method;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import org.jspecify.annotations.Nullable;
+import org.openrewrite.*;
+import org.openrewrite.java.AnnotationMatcher;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.service.AnnotationService;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.TypeUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+public class RemoveUnnecessaryOverride extends Recipe {
```

Comment:
Did you know we already have a [RemoveImplements recipe](https://github.com/openrewrite/rewrite/blob/de210f8d4ec60ec4d01ebbac519e0bfa4f4a64b5/rewrite-java/src/main/java/org/openrewrite/java/RemoveImplements.java#L33) as well? Seems like that might be a good option here.

Reply:
Had a closer look just now ; turns out we didn't yet need the option to remove implements, so I've stripped that down and suggest we first look at the existing recipe if the need arises in subsequent work here, thanks!